styles.css

```diff
@@ -0,0 +1,285 @@
+
+.header > .list {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+.firstsection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 250px;
+  font-weight: bold;
+
+}
+
+#home-page-title {
+  display: flex;
+  justify-content:center;
+  font-size: 40px;
+}
+
+.button {
+    border: 1px solid black;
+    border-radius: 4px;
+    color: black;
+    padding: 20px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-weight: normal;
+    font-size: 16px;
+    margin: 25px;
+}
+
+.secondsection {
+  display: flex;
+  flex-direction: row;
+  font-style: italic;
+  padding-left: 10%;
+}
+
+blockquote:before {
+  content: open-quote;
+}
+
+blockquote:after {
+  content: close-quote;
+}
+
+.thirdssection {
```

Comment:
Nice padding! You chose to use display: block - what made you decided to do this over flex or grid?